localnet/poktrolld/config/app.toml

```diff
@@ -49,10 +49,13 @@ query-gas-limit = "0"
     stop-node-on-err = true
 
 [telemetry]
+  datadog-hostname = ""
```

Comment:
We're going with datadog?

Reply:
@Olshansk we don't - this is a default config file that was created by ignite.